### ISSUE TITLE
Should DCA be calculated using training data? 

### ISSUE THREAD

**Issue by PaulHiemstra** [MSKCC-Epi-Bio/decisioncurveanalysis#15]
To my knowledge, DCA provides an alternative to other metrics in the form of the net benefit. Typically, this kind of evaluation metric is calculated on data the fitted model has not yet seen to provide a more honest insight into the performance of the model. This can for example be done using train test split or cross validation.

I noticed that the tutorial for DCA does not do this. Is there a reason why? Does net benefit not suffer from the above mentioned problem?

I have done some tests, and the outcome is roughly similar, but not completely so.

**Comment by VickersA** [comment]
Issues of overfit and optimism are exactly the same for net benefit as for other performance metrics. See https://pubmed.ncbi.nlm.nih.gov/19036144/ for our discussion in the literature and https://mskcc-epi-bio.github.io/decisioncurveanalysis/dca-tutorial.html#Correction_for_Overfit for the software tutorial.

**Comment by PaulHiemstra** [comment]
I was not interested in the survival analysis as such, and had nog scrolled down far enough. My apologies, I should have seen this. Thanks for taking the time to provide feedback!
